.gitmodules

```diff
@@ -124,6 +124,9 @@
 [submodule "projects/multicenter-phantom"]
 	path = projects/multicenter-phantom
 	url = https://github.com/conpdatasets/multicenter-phantom.git
+[submodule "projects/Bigbrain_3DROIS"]
+	path = projects/Bigbrain_3DROIS
+	url = https://github.com/mathdugre/conp-dataset-Bigbrain-3DROIS.git
```

Comment:
Instead of having the dataset in your username, could you publish the DataLad dataset to https://github.com/conpdatasets/BigBrain_3DROIs instead and link that repo in the submodule?

We try avoiding to have datasets hosted on our own accounts as much as possible.

Reply:
I just gave you write access to the repo so you can publish directly to it via DataLad.

Maybe the reason the repo cannot be forked is because it is empty?

Reply:
I brought the change to the submodule url.